dashboard/modules/job/tests/test_job_manager.py

```diff
@@ -238,11 +253,24 @@ def dict_to_str(d):
         wait_for_condition(
             check_job_succeeded, job_manager=job_manager, job_id=job_id)
         assert dict_to_str({
+            JOB_NAME_METADATA_KEY: job_id,
             JOB_ID_METADATA_KEY: job_id,
             "key1": "val1",
             "key2": "val2"
         }) in job_manager.get_job_logs(job_id)
```

Comment:
I think it's worth considering adding a "get_metadata" API for jobs in this case, if user wants to submit a job with tags for experiment management / filtering.